pkg/backup/http.go

```diff
@@ -69,6 +70,11 @@ func (b *Backup) serveSnap(w http.ResponseWriter, r *http.Request) {
 		http.NotFound(w, r)
 		return
 	}
+	if v1, v2 := r.FormValue("version"), getEtcdVersionFromSnap(fname); v1 != v2 {
```

Comment:
we need to use semvar(https://github.com/coreos/go-semver) to parse the etcd version and find if the latest backup is OK for that version.

we can keep a version map here to check from

[3.0]{2.3,3.0}
[3.1]{3.0,3.1}
...